sdk/Sdk.Analyzers/TypeSymbolExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Azure.Functions.Worker.Sdk.Analyzers
+{
+    internal static class TypeSymbolExtensions
+    {
+        internal static bool IsAssignableFrom(this ITypeSymbol targetType, ITypeSymbol sourceType, bool exactMatch = false)
```

Comment:
I see that this method is only called once and with `exactMatch = true` (unless I missed occurrences). Wondering if this method is required, as the only code-path we ever go is to check if target and source are exact match.

Reply:
Want to keep this as it it will be useful in a lot of analyzers.